repos/system_upgrade/el8toel9/actors/dotnet/tests/test_dotnet.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from leapp.models import InstalledRedHatSignedRPM, Report, RPM
+from leapp.snactor.fixture import current_actor_context
```

Comment:
I don't understand this error:
```
F401 'leapp.snactor.fixture.current_actor_context' imported but unused
```
As far as I can tell, I'm using it in a similar way as other tests.